docs/tutorials/writing_docs.rst

```diff
@@ -0,0 +1,244 @@
+- The purpose of every sentence in documentation should be to convey relevant
+  information.
+  - No fluff.
+  - No sales pitches.
+  - Save the jokes for code comments.
+- Use links to tie in additional concepts, rather than re-explaining.
+- If two concepts differ only slightly, fully document one. For the second,
+  refer to the first and note the differences.
+
+.. _Consistent:
+
+Consistent
+----------
+
+All of the documentation should be written in the same basic voice. This
+includes using a consistent tone, tense, style, and word choices.
+`Google's developer documentation style guide <https://developers.google.com/style/tone>`_ thoroughly covers this
+subject, and should be considered the the style guide for Pavilion
+documentation as well.
+
+The downside of it being thorough is that it is large. Here are some parts
+that are particularly important:
+
+- `Style and Tone <https://developers.google.com/style/tone>`_
+- `Present Tense <https://developers.google.com/style/tense>`_
+- `Pronouns <https://developers.google.com/style/pronouns>`_
+- `Second Person <https://developers.google.com/style/person>`_
+
+Structure
+^^^^^^^^^
+
+Structure each document fractally. Each document should have a well defined
+subject, divided into sub-sections with well defined subjects, that are
+themselves sub-divided either into further sub-sub-sections or paragraphs.
+
+While the above may seem trivial, it forms the foundation for how to approach
+writing documentation.
+
+1. Very briefly summarize the document in the header.
+2. Outline the sub-topics by creating section headers.
+3. Write an introduction/overview that ties the sub-topics together.
+4. Write each sub-topic as if it is its own document, starting from step 1.
+
+   - When the ideas are too small for sub-sections, then it is time to
+     write paragraphs.
+   - If the ideas in a document require more than 4 levels of headings,
+     you may need to consider moving topics to their own documentation pages.
+   - In contrast, document length should not be a reason to break up a page.
+   - For very small ideas with little content, use a dictionary list.
+
+
+Headings
+````````
+Headings should tell the user precisely what the section is about, given
+the context of the heading structure. You'll have a table of contents
+to reinforce this hierarchy.
+
+- Bad example: "Writing the source". What source?
+- Better example: "Writing a result parser module". That module.
+- Best example: "Result parser module" within a "Plugin Files" section in a
+  tutorial document about writing result parser plugins. We
+  know by context this a plugin file and that we'll be writing it.
+
+Documentation Type
+``````````````````
+There are four types of documentation within Pavilion:
+
+Usage
+  The purpose of usage documentation is to give the reader a quick
+  introduction to the overall structure of Pavilion. It should not try to
+  completely document any subject, and should instead link to the **Feature**
```

Comment:
'It should not try to completely document any subject; instead, it should link to the **Feature** documentation on that topic.'
Granted, I'm awful at using semicolons, but the [compound sentences](https://academicguides.waldenu.edu/writingcenter/grammar/sentencestructure) section seems to suggest that this is a good time to use it.  Up to you.